velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -281,6 +281,9 @@ std::unique_ptr<ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
     for (int32_t i = 0; i < schemaElement.num_children; i++) {
       ++schemaIdx;
       auto& childName = schema[schemaIdx].name;
```

Comment:
Let's remove the reference since we could be modifying the value.
`auto childName = `